packages/cli/src/utils.ts

```diff
@@ -230,6 +234,33 @@ export function printLintTotals(totals: Totals, definitionsCount: number) {
   process.stderr.write('\n');
 }
 
+export function printConfigLintTotals(totals: Totals): void {
+  if (totals.errors > 0) {
+    process.stderr.write(
+      red(
+        `❌ Your lint config has ${totals.errors} ${pluralize('error', totals.errors)}${
+          totals.warnings > 0
+            ? ` and ${totals.warnings} ${pluralize('warning', totals.warnings)}`
+            : ''
+        }.\n`,
+      ),
+    );
+  } else if (totals.warnings > 0) {
+    process.stdout.write(
+      green(`Your lint config is valid. 🎉\n`),
```

Comment:
```suggestion
      green(`Your config is valid. 🎉\n`),
```

Reply:
We probably don't need to greet a user if the config is valid at all :) The main point is to have meaningful warnings when the config is invalid, am I right?
BTW @RomanHotsiy, should it be `stdout` where we put the messages? I remember once this broke something in **Workflows**.

Reply:
I would not show a "valid" message too. Only warnings or errors.

We should use `stderr` for all the user logs (even not errors). Stdout is used for the actual tool output (bundle or lint results JSON).